common/libs/wallet/base.js

```diff
@@ -4,4 +4,8 @@ export default class BaseWallet {
   getAddress(): string {
     throw 'Implement me';
   }
+
+  getNakedAddress(): string {
+    return this.getAddress().replace('0x', '').toLowerCase();
```

Comment:
yeah, so by default we use mixed case for checksums, this one is used as eth_call argument, not sure if significant - ported from v3.